tests/test_jobs/test_background_job_manager.py

```diff
@@ -36,23 +36,23 @@ class TestBackgroundJobManager:
     def test_start_mock_jobs(self):
         """Runs jobs as scheduled and skips disabled jobs."""
         with override_config(app, 'JOB_MANAGER', _job_manager_config()):
-            job_manager = BackgroundJobManager()
+            job_manager = BackgroundJobManager(
+                available_job_classes=[
+                    DoomedToFailure,
+                    HelloWorld,
+                    LightSwitch,
+                    Volume,
+                    VolumeSubClass,
+                ],
+            )
             try:
+                def _get_job_history_ids():
+                    return [item.id for item in JobHistory.get_job_history_in_past_days(day_count=1)]
+
+                original_job_history_ids = _get_job_history_ids()
                 job_manager.start(app)
-                # The JOB_MANAGER config has four (4) jobs but one is disabled. So, only three are loaded.
-                assert len(job_manager.job_instances) == 3
                 time.sleep(2)
-
-                for job_instance in job_manager.job_instances:
-                    if job_instance.key() == 'volume':
-                        assert job_instance.level == 11
-                    elif job_instance.key() == 'volume_sub_class':
-                        # Property not yet set
-                        assert job_instance.level is None
-                    elif job_instance.key() == 'light_switch':
-                        assert job_instance.is_light_on is True
-                    else:
-                        assert False
+                assert len(_get_job_history_ids()) == len(original_job_history_ids) + 2
```

Comment:
The assertions above go away because I've tossed out `job_manager.job_instances` property. IMO, verifying the count is good enough.